### DISCUSSION TITLE
Problem with sending documents and location.

### DISCUSSION THREAD

**Discussion by leandrolombaldo** [community/community#113574] [GitHub Education] [answered]
### Hi everyone,

I need some help, I haven't obtained my student ID yet, and I'm studying remotely. When I try to send the documents, there's only an option to take a photo, but no option to upload. My university is located in another state, so it's not recognized as nearby. Does anyone have any tips to solve these problems? Thank you!

**Comment by byzaimimtiaz** [comment]
I am facing same error, if anyone know the solution, kindly let me know.

**Comment by samus-aran** [comment, marked as answer]
Hi @leandrolombaldo 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I have a double-sided Student ID with the date on the other side. How do I submit this as evidence?](https://github.com/orgs/community/discussions/111352#double-sided-student-id-with-the-date-on-the-other-side)

I want to highlight an important part of this: **You may include multiple documents in your image, so long as they are legible. So as an example you could take a photo of your student ID card alongside a dated class schedule.**

Ensure your evidence includes important information e.g. your name, school etc.

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍

**Reply by sreenaveen-07** [reply]
I have the same problem. I have come home(far from my college's location) for a break due to some health issue, due to which,I won't be able to return to my college hostel for another few weeks- during which I'm doing an informal distance education. What should i submit as a proof for staying at home?
